tests/system/providers/google/cloud/cloud_run/example_cloud_run_service.py

```diff
@@ -0,0 +1,106 @@
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+"""
+Example Airflow DAG that uses Google Cloud Run Service Operators.
+"""
+
+from __future__ import annotations
+
+import os
+from datetime import datetime
+
+from google.cloud.run_v2 import Service
+from google.cloud.run_v2.types import k8s_min
+
+from airflow.models.dag import DAG
+from airflow.operators.python import PythonOperator
+from airflow.providers.google.cloud.operators.cloud_run import (
+    CloudRunCreateServiceOperator,
+    CloudRunDeleteServiceOperator,
+)
+
+PROJECT_ID = os.environ.get("SYSTEM_TESTS_GCP_PROJECT", "default")
+ENV_ID = os.environ.get("SYSTEM_TESTS_ENV_ID")
+DAG_ID = "example_cloud_run_serivce"
+
+region = "us-central1"
+service_name_prefix = "cloudrun-system-test-service"
+service_name = f"{service_name_prefix}"
+
+create_task_name = "create-service"
+delete_task_name = "delete-service"
+
+
+def _assert_created_services_xcom(ti):
+    service1_dicts = ti.xcom_pull(task_ids=[create_task_name], key="return_value")
+    assert service_name in service1_dicts[0]["name"]
+
+
+# [START howto_create_service_instance]
+def _create_service():
```

Comment:
I think I misunderstood something, can you please explain why we need to create a service in this method here if we have operators to do the same? Doc strings for this CreateOperator tell that we are creating new service, am I wrong?

Reply:
no, i was wrong.. (ref: [docs](https://cloud.google.com/python/docs/reference/run/latest/google.cloud.run_v2.services.services.ServicesClient#google_cloud_run_v2_services_services_ServicesClient_create_service))
if request(CreateServiceRequest) is provied, service should not be set.
and i deleted param `service` in create/delete_service

Reply:
> no, i was wrong.. (ref: [docs](https://cloud.google.com/python/docs/reference/run/latest/google.cloud.run_v2.services.services.ServicesClient#google_cloud_run_v2_services_services_ServicesClient_create_service)) if request(CreateServiceRequest) is provied, service should not be set. and i deleted param `service` in create/delete_service

I didn't understand your last statement - in which methods was `service` deleted? It seems that you initalize a service object in the next line (56)

Reply:
@shahar1 @VladaZakharova 
I was wrong... 🫠🫠

In the parameter description of [create_service](https://cloud.google.com/python/docs/reference/run/latest/google.cloud.run_v2.services.services.ServicesClient#google_cloud_run_v2_services_services_ServicesClient_create_service), **it indicates `Service` instance should be required.** I need to modify it's currently written..🥲 need to initialize a service instance in the hook like [cloud_run job](https://github.com/apache/airflow/blob/6229c1c98a2a78d00255d6b6c5b70d032cff9b80/airflow/providers/google/cloud/hooks/cloud_run.py#L88) and add it to param.

I'll fix it soon, push it, mention it, and report my local test result to yours!

Reply:
@shahar1 any followup on this thread?